manifests/manage_keys.pp

```diff
@@ -13,7 +14,7 @@
   ssh_authorized_key { $key_title:
     ensure => present,
     user   => $user,
-    name   => $key_name,
+    name   => "${user}_${key_name}",
```

Comment:
It's only necessary because the original code included the `name` attribute and I didn't want to remove it, although I'm not super sure why at this point (it's been a bit). See: https://docs.puppet.com/puppet/latest/reference/types/ssh_authorized_key.html#ssh_authorized_key-attribute-name

So if `name` is included then it needs to be system-wide unique, which I did by just prepending the the username. However, `$key_title` should already be unique, so there wasn't really any reason to include the `name` attribute at all since `$key_title` as the resource name should be fine.

The more correct/compact definition should probably just be:
```
ssh_authorized_key { $key_title:
  ensure => present,
  user   => $user,
  key    => $key_content,
  type   => $key_type,
  target => $key_file,
}
```

I'll do a quick test and update the PR.